pedsnetdcc/abstract_transform.py

```diff
@@ -28,13 +28,16 @@ class Transform(object):
     __metaclass__ = ABCMeta
 
     @classmethod
-    def pre_transform(cls, conn_str):
+    def pre_transform(cls, conn_str, metadata=None, table_name=None):
```

Comment:
I just got this totally wrong. The `table_name` is not needed. The `IDMappingTransform` just needs to do the pre-transformation work for the whole database (all tables) when it is called.